_docs/setup/kubernetes/helm.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Alpha Quality Helm Chart
+overview: Quick Start instructions for the setup and configuration of Istio using the Helm package manager.
+
+order: 10
+
+layout: docs
+type: markdown
+---
+
+{% include home.html %}
+
+Quick Start instructions for the setup and configuration of Istio using the Helm package manager.
+
+
+## Prerequisites
+
+The following instructions require you have access to Helm **2.7.2 or newer** in your Kubernetes environment or 
+alternately the ability to modify RBAC rules required to install Helm.
```

Comment:
Let us refer to k8s version in the setup doc too.

Reply:
fixed.